README.md

```diff
@@ -1 +1,29 @@
-This is a serverextension and lab extension for running LaTeX from inside the jupyterlab interface.
+# JupyterLab Latex
+
+An extension for JupyterLab which allows for live-editing of LaTeX documents.
+
+To use, right-click on a `.tex` document within JupyterLab, and select `Show LaTeX Preview`.
+This will compile the `.tex` file and open the rendered PDF document.
+Subsequent saves of the file will automatically update the PDF.
+If the PDF fails to compile (possibly due to a syntax error),
+an error panel will open detailing the LaTeX error.
+
+## Requirements
+* JupyterLab 0.29
+* Something that can compile `.tex` files to PDF on your notebook server (e.g., `pdflatex` or `xelatex`).
+
+## Installation
+
+In order to enable this extension you must install both a notebook server extension and a lab extension.
```

Comment:
I would be more specific here as to exactly what is the notebook server extension and what is the lab extension or are they both in this repo?

Reply:
They are both in this repo. I have added some text clarifying.